prowler/lib/outputs/json.py

```diff
@@ -69,11 +74,14 @@ def fill_json_asff(finding_output, audit_info, finding, output_options):
         associated_standards = []
         check_compliance = get_check_compliance(finding, "aws", output_options)
         for key, value in check_compliance.items():
-            associated_standards.append({"StandardsId": key})
-            item = f"{key} {' '.join(value)}"
-            if len(item) > 64:
-                item = item[0:63]
-            compliance_summary.append(item)
+            if (
+                len(associated_standards) < 20
+            ):  # AssociatedStandards should NOT have more than 20 items
+                associated_standards.append({"StandardsId": key})
+                item = f"{key} {' '.join(value)}"
+                if len(item) > 64:
+                    item = item[0:63]
+                compliance_summary.append(item)
```

Comment:
Could you include this as a part of the class validator? It's so easy to do it with Pydantic https://docs.pydantic.dev/latest/usage/validators/